### DISCUSSION TITLE
Android app doesn't have bottom bar

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44240] [Mobile] [answered]
I was comparing GitHub Mobile beta for Android with a coworker's GitHub Mobile app for iOS.

Here's a screenshot (Slack Android) of a GitHub Mobile iOS screenshot for a pull request review:
![Screenshot_20230113-112051.png](https://user-images.githubusercontent.com/2119212/212368567-4a1020c0-fb24-461a-9cc5-b41c1dd26693.png)

On Android, the bottom bar is generally not available, e.g. as seen here:
- https://github.com/community/community/discussions/44233
It's only available on the main screen, which isn't particularly helpful. 

This is somewhat related to:
- https://github.com/community/community/discussions/43959

**Comment by stevepopovich** [comment]
Hi @jsoref I would argue here that this is by design. We navigate to many screens that need more real estate for what we are showing on the screen. I would also argue here that this is typical and consistent Android behavior that users would expect. Thanks for the feedback!

**Reply by Syndelis** [reply]
This does make navigating around the app harder. If you were inside a file of repository A, and now wants to check repository B, you either spam the "go back" button, or re-open the app (which should not be faster IMO)

**Reply by jsoref** [reply]
Gmail has a bottom bar:
![Gmail for Android with a GitHub community discussion](https://user-images.githubusercontent.com/2119212/225332185-c152a83b-3db8-4298-8718-1a4b37175cbb.png)

Contacts has a bottom bar:
![Contacts for Android](https://user-images.githubusercontent.com/2119212/225332203-4395f6c6-8234-45ad-8c04-1218ced7860a.png)

Files has a bottom bar:
![Files for Android](https://user-images.githubusercontent.com/2119212/225332220-67ed04aa-1f04-42f3-ab74-7c259f75d488.png)

**Reply by jsoref** [reply]
@elisealix22 I don't suppose I could entreat you to file a ticket for this? :)

**Reply by stevepopovich** [reply, marked as answer]
Hi @jsoref we are actually already tracking this internally and are working on the ideation of a better experience for this. Thanks for writing in!

**Reply by TuxVinyards** [reply]
The app continues to improve, which is good.  :+1: 

But **_people are still being expected the have to force close the app_** just to get back to to their home start page.   

How hard is it to have a 'home' link underneath where it says 'share'  in the top right menu?

**Reply by stevepopovich** [reply]
HI @TuxVinyards fair feedback. We are still tossing this idea around internally and are playing with prototypes and design. We will let you know when we have made significant changes to the bottom bar! For now, you should always have the back button to be able to navigate back to home through the backstack.

**Reply by TuxVinyards** [reply]
@stevepopovich  Thanks for reply.  

"For now, you should always have the back button to be able to navigate back to home through the backstack."

**_This is nub of the problem._**   If go look at someone elses repo and then at the code, followed etc etc you can end with as many  40 odd back presses that our needed....   It gets a bit ridiculous.

**Reply by stevepopovich** [reply]
That's fair. Browsers do it better and other apps do it better. We will continue to work towards a better solution!

**Comment by d-kavinraja** [comment]


**Comment by ZXMLSLYJ** [comment]
很多圈子都有一种现象，想着是眼界大点格局太小的圈内中间人凸显自己懂了一点的优越感的行为
比如就是飞马在跑圈的地位，狩猎者在笔圈的地位，白夜行在推理小说的地位，三体在科幻小说的地位……
就这些人的嘴脸像极了那些仗着"过来人"的身份对我们人生指指点点的亲戚，比来比去，鄙来鄙去，不累吗，不能纯粹一点吗
不是靠这些破圈的东西，能有多少新人？非要让这个圈子就剩你和你的好友，这么自私么，封建制灭了多少年了。
包容一点好嘛？谁不是小白来的？产品的好坏，只有时间能最客观的评价。
